pages/linux/ufw.md

```diff
@@ -11,26 +11,26 @@
 
 `ufw disable`
 
-- Show ufw rules along with their numbers:
+- Show ufw rules, along with their numbers:
 
 `ufw status numbered`
 
 - Allow incoming traffic on port 5432 on this host:
 
 `ufw allow {{5432}}`
 
-- Allow traffic from 192.168.0.4 to any address on this host on port 22 using only tcp protocol:
+- Allow traffic from 192.168.0.4 to any address on this host, on port 22, using only the tcp protocol:
 
 `ufw allow from {{192.168.0.4}} to {{any}} port {{22}} proto {{tcp}}`
 
 - Deny traffic on port 80 on this host:
 
 `ufw deny {{80}}`
 
-- Deny traffic to port 22 using only udp protocol:
+- Deny traffic to port 22, using only the UDP protocol:
```

Comment:
The combined example does seem to put on a bit of cognitive load to completely grasp the pattern. I am okay with changing the description. Agree with @ostera's thinking.